README.md

```diff
@@ -1,6 +1,208 @@
+
+##### list the related cohort encounter {#CohortModule-listtherelatedcohortencounter}
+
+GET: [http://localhost:8080/openmrs/ws/rest/v1/cohortm/cohortenc/47d7bb8c-c4f2-4d16-978a-a4c7cf0d9556](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortenc/47d7bb8c-c4f2-4d16-978a-a4c7cf0d9556)
+
+##### list the related cohort observation {#CohortModule-listtherelatedcohortobservation}
+
+GET : [http://localhost:8080/openmrs/ws/rest/v1/cohortm/cohortobs/34839160-5c8a-4c37-bdf6-e17042d46614](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortobs/34839160-5c8a-4c37-bdf6-e17042d46614)
+
+***
+
+#### Example POST Requests
+
+POST: [http://localhost:8080/openmrs/ws/rest/v1/cohortm/cohort](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohort/2baddb64-f4f9-47c4-80c6-404c4b016d12)
+
+PARAMETERS:
+
+{"name":"neurologicalrecoverystudy","cohortType":"\(list the related cohortType uuid\)","cohortProgram":"\(list the related cohortProgram uuid\)"}
+
+
+
+POST :[http://localhost:8080/openmrs/ws/rest/v1/cohortm/](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortmember/2003394e-daa2-4df8-acba-4894460b0624)
+
+[PARAMETERS ](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortmember/2003394e-daa2-4df8-acba-4894460b0624)
+
+{"name":"Horatio","person":"\(list the related person uuid\)","cohort":"\(list the related cohort uuid\)"}
+
+
+
+POST: [http://localhost:8080/openmrs/ws/rest/v1/cohortm/](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortenc/47d7bb8c-c4f2-4d16-978a-a4c7cf0d9556)
+
+[PARAMETERS ](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortenc/47d7bb8c-c4f2-4d16-978a-a4c7cf0d9556)
+
+{"encounterType":"\(list the related encounterType uuid\)","visit":"\(list the related visit uuid\)","cohort":"\(list the related cohort uuid\)","form":"\(list the related form uuid"\)}
+
+
+
+POST: [http://localhost:8080/openmrs/ws/rest/v1/cohortm/](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortenc/47d7bb8c-c4f2-4d16-978a-a4c7cf0d9556)[cohortobs](http://localhost:8080/openmrs/ws/rest/v1/cohortm/testcohortobs/34839160-5c8a-4c37-bdf6-e17042d46614)
+
+PARAMETERS{"encounterId":"\(list the related encounter uuid\)","concept":"\(list the related concept uuid\)","cohort":"\(list the related cohort uuid\)"}
+
+***
+
+* The min version required for the OpenMRS Core is 1.11.0
+
+* The two modules Rest Module of 2.11+ version and HTML Form Entry of 2.5+ version are the required modules.
+
+* **HTML FORM ENTRY OMOD FILE**: [https://modules.openmrs.org/\#/show/58/htmlformentry](https://modules.openmrs.org/#/show/58/htmlformentry)
+
+* **REST MODULE OMOD FILE**: [https://modules.openmrs.org/\#/show/153/webservices-rest](https://modules.openmrs.org/#/show/153/webservices-rest)
+
+* These modules should be running on OpenMRS core before installing the Cohort Module.
+
+#### Video for Installation Of The Module
```

Comment:
Remove this section, it includes an older version. I'm gonna put up a task to demo the new cohort module